src/App.tsx

```diff
@@ -1,23 +1,51 @@
-import React from 'react';
+import { FC, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tab } from './types';
+import { Tabs } from './components/Tabs/Tabs';
 
-export const tabs = [
+export const allTabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [tabs] = useState<Tab[]>(allTabs);
+  const [selectedTabId, setSelectedTabId] = useState<string>('');
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
```

Comment:
you can get the info by using `useMemo` hook based on `tabs` and `selectedTabId`